kubevirt/step1.md

```diff
@@ -0,0 +1,49 @@
+### Wait for the cluster to be ready
+
+Before we can start, we need to wait for the Kubernetes cluster to be ready
+
+#### Deploy KubeVirt
+
+Deploy KubeVirt operator using latest Kubevirt Version
+
+`export KUBEVIRT_VERSION=$(curl -s https://api.github.com/repos/kubevirt/kubevirt/releases|grep tag_name|sort -V | tail -1 | awk -F':' '{print $2}' | sed 's/,//' | xargs)
```

Comment:
We had it with JQ, and moved it to this approach to use exactly the same snippet we've at the website (and what our prow CI tests) (in fact my PR on https://github.com/kubevirt/kubevirt.github.io/pull/243 was using jq)

Whatever choice is good, but it should be 'standard' on all the docs we use

Reply:
I kind of disagree that the command must be the same as in other places. The reason `jq` isn't ideal in the website is because it's not as widely available as sed and awk, but the Katacoda environment does have `jq` available and it makes things simpler.

I'd value simplicity over consistency with other labs/demos, but it's up to you.

In any case it might not hurt to briefly explain what this does, again with the goal of helping people understand the contents of the lab.

Reply:
thanks, let me then go back to jq and put some more indications about each steps, goals, etc